docs/environment_variables.md

```diff
@@ -50,8 +50,10 @@ The RawProfilerHook allows one additional profiler that has not yet on-boarded t
 
 | Variable | Value | Description |
 |-|-|-|
-MicrosoftInstrumentationEngine_RawProfilerHook|"{GUID}"|This would be the value set to CORECLR/COR_PROFILER.
-MicrosoftInstrumentationEngine_RawProfilerHookPath_32/64|"[FULL PATH TO raw profiler dll]"|This would be the value set to CORECLR/COR_PROFILER_PATH_32/64.
+MicrosoftInstrumentationEngine_RawProfilerHook|"{GUID}"|This would be the value set to COR_PROFILER.
+MicrosoftInstrumentationEngine_CoreRawProfilerHook|"{GUID}"|This would be the value set to CORECLR_PROFILER. Falls back to MicrosoftInstrumentationEngine_RawProfilerHook if this is not set.
```

Comment:
So this solves the situation where someone wants a raw profiler for coreclr, but not desktop clr. The opposite scenario is not possible (without a breaking change).

Reply:
Correct, we could introduce an additional separate desktop-specific RPH variable as well, that would not break anything.

Reply:
Or if a data collector wants to blindly specify both CLR and CoreCLR profiler information and let the runtime decide what profiler to load.